util/topgen/templates/toplevel.sv.tpl

```diff
@@ -505,7 +505,7 @@ mem_name = lib.Name(mem_name[1:])
     .ErrOnWrite(1),
     .CmdIntgCheck(1),
     .EnableRspIntgGen(1),
-    .EnableDataIntgGen(1) // TODO: Needs to be updated for intgerity passthrough
+    .EnableDataIntgGen(0)
```

Comment:
We'll have to update the `sram_ctrl` testbench as well with this change:
https://github.com/lowRISC/opentitan/blob/d578ec73fa2a6d4c588b0c955b16fa5a3108476b/hw/ip/sram_ctrl/dv/sram_ctrl_wrapper.sv#L94-L117

@udinator @weicaiyang is the bus integrity automatically enabled in all our testbenches?

Reply:
ah yes thanks for the reminder.

Reply:
@msfschaffner you mean in any tl_adapter_sram instances?

Reply:
looks like we should enable data integrity gen here as well as enable the check in rv_core_ibex

Reply:
Oh no, just the instances that are used in the `sram_ctrl` testbench.
Other IPs that make use of the `tlul_adapter_sram` don't have to perform this read-modify-write operation, since they do not store ECC alongside the data in the local memories.

Reply:
got it. no, it's not automatically enabled in the sram testbench, but I believe Tim will enable it, see #7177